cmd/genz/genz.go

```diff
@@ -67,9 +69,23 @@ func (c generateCommand) Run() error {
 		tags = strings.Split(*buildTags, ",")
 	}
 
-	template, err := os.ReadFile(*templateFile)
-	if err != nil {
-		return fmt.Errorf("failed to read template file %s: %v", *templateFile, err)
+	var template []byte
+	if strings.HasPrefix(*templateFile, "http://") || strings.HasPrefix(*templateFile, "https://") {
+		response, err := http.Get(*templateFile)
```

Comment:
Maybe use `url` package to properly check that `templateFile` is a correct URL.
```suggestion
                url, err := url.ParseRequestURI(*templateFile)
	        if err != nil {
		    return "", fmt.Errorf("failed to parse template file url %s: %v", *templateFile, err)
	        }
	        response, err := http.Get(url.String())
```